File: Java/01.Java-Basic/07.ForLoop/src/forLoop/exercise/Salary.java
package forLoop.exercise;

import java.util.Scanner;

public class Salary {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int facebookPrice = 150;
        int instagramPrice = 100;
        int redditPrice = 50;

        int tabsSize = Integer.parseInt(scanner.nextLine());
        double salary = Double.parseDouble(scanner.nextLine());
        for (int i = 0; i < tabsSize; i++) {
            String site = scanner.nextLine();
            switch (site) {
                case "Facebook":
                    salary -= facebookPrice;
                    break;
                case "Instagram":
                    salary -= instagramPrice;
                    break;
                case "Reddit":
                    salary -= redditPrice;
                    break;
                default:
                    break;
            }

            if (salary <= 0) {
                System.out.println("You have lost your salary.");
                break;
            }
        }

        if (salary > 0) {
            System.out.printf("%.0f", salary);
        }
    }
}

File: JavaScript/01.JavaScriptBasic/08For-Loop-Exercise/src/Salary.js
function fine(params) {
    let tabs = Number(params[0]);
    let salary = Number(params[1]);
    for (let index = 0; index < tabs; index++) {
        const element = params[index + 2];
        switch (element) {
            case "Facebook":
                salary -= 150;
                break;

            case "Instagram":
                salary -= 100;
                break;

            case "Reddit":
                salary -= 50;
                break;

            default:
                break;
        }

        if (salary <= 0) {
            console.log("You have lost your salary.");
            break;
        }
    }

    if (salary > 0) {
        console.log(salary);
    }
}

fine(["10",
"750",
"Facebook",
"Dev.bg",
"Instagram",
"Facebook",
"Reddit",
"Facebook",
"Facebook"]);